electron/electron.ffmpeg.js

```diff
@@ -0,0 +1,83 @@
+/*
+    FFMpeg wrapper
+*/
+
+import log from 'electron-log';
+import { spawn } from 'child_process';
+import { main } from '../config/config';
+
+const { rtmpPort, frameConfig } = main;
+
+/* track if user stopped the process */
+let userStopFFmpeg = false;
+
+const startFFMpeg = (sender, rtmpStrmID, configIdx = 0) => {
+    log.info(`Launching FFmpeg with config: ${configIdx}`)
+    userStopFFmpeg = false  /* reset !!*/
+
+    const framerate = frameConfig[configIdx].framerate
+    const keyint = frameConfig[configIdx].keyint
+    const FFMPeArgs = [
+        '-f', 'avfoundation',
+        '-framerate', framerate,
+        '-pixel_format', 'uyvy422',
+        '-i', '0:0',
+        '-vcodec', 'libx264',
+        '-tune', 'zerolatency',
+        '-b', '900k',
+        '-x264-params', `keyint=${keyint}:min-keyint=${keyint}`,
+        '-acodec', 'aac',
+        '-ac', '1',
+        '-b:a', '96k',
+        '-f', 'flv',
+        `rtmp://localhost:${rtmpPort}/stream/${rtmpStrmID}`];
+
```

Comment:
Yeah - the endpoints have changed a little bit.  @buildog are you changing the app to match the new endpoints?  

I think basically we are missing 3 endpoints in the new node:
`/peersCount`
`/createStream`
`/localStreams`

I'm adding them back so the app can still get the information.